src/products/firefox_desktop/adding_release.md

```diff
@@ -44,6 +44,44 @@ The language name associated to the locale code is displayed in a few places in
 
 This is an [example of a bug](https://bugzilla.mozilla.org/show_bug.cgi?id=1807794) created to add this information. Don’t forget to update the GitHub tracking issue for the locale with a link to this bug.
 
+## Verify locale-specific settings
+
+Some of the locale-specific settings are defined inline in Firefox code, and should be reviewed if they need customization for your locale:
```

Comment:
`your locale` doesn't make sense in the context of this document (across this update), since the target is Localization EPMs. `the locale`, `the new locale`, etc. work.